packages/Liquid-Core.package/LQPoll.class/instance/exportCSV.st

```diff
@@ -1,8 +1,12 @@
 printing
 exportCSV
-
-	| stream |
-	
-	stream := WriteStream on: String new.
-	self printDataOn: stream withDelimiter: ';'.
-	^ stream contents
\ No newline at end of file
+	| filePath directory |
+	directory := FileDirectory on: self class pathToExport.
+	directory exists
+		ifFalse: [FileDirectory default createDirectory: self class pathToExport].
+	filePath := self class pathToExport , FileDirectory default pathNameDelimiter , self id , '.csv'. 
+	(FileDirectory default fileExists: filePath)
+		ifTrue: [FileDirectory default deleteFileNamed: filePath].
```

Comment:
I'm not a fan of deleting a file without asking the user for confirmation or making it abundantly clear that this will happen. Neither is currently happening.

I would prefer to invoke the default Squeak menu for selecting the preferred action, but a simple confirmation dialog will be fine.

Check out the [Data Sync Code Example File](https://moodle.hpi.de/mod/resource/view.php?id=26570) for details on adapting the tests accordingly.